### PR TITLE
Show info message to report any errors if validation was run by addons-linter.

### LINKED ISSUE #2627: Show info message that if the validation output comes from addons-linter
We have those little notification boxes in the preliminary validation result screen once you submit an add-on. Let's use those boxes to show the users a small info message like "Hey, we're trying out a new linting library, if there are any errors please report them here ..."

### PR DESCRIPTION
**Pull request by EnTeQuAk**
Fixes #2627.

This doesn't neccessarily show it in all cases but at least for regular error-cases.

screenshot from 2016-05-31 07-39-01

### CODE REVIEW

**Comment by diox** [comment]
r+

File: static/js/common/upload-addon.js [hunk 1]
```diff
@@ -191,6 +191,14 @@
                 upload_progress_outside.attr('class', 'bar-fail');
                 upload_progress_inside.fadeOut();
 
+                if (results && results.processed_by_addons_linter) {
+                    $("<a>").text(gettext('We have enabled a new linter to process your Add-on. Please make sure to report any issues on GitHub'))
```
Comment (andymckay) [inline static/js/common/upload-addon.js]: Add-on should not be capitalized.

File: static/js/common/upload-addon.js [hunk 2]
```diff
@@ -191,6 +191,14 @@
...
+                            .attr('href', 'https://github.com/mozilla/addons-linter/')
+                            .attr('class', 'addons-linter-info')
+                            .attr('target', '_blank')
```
Comment (andymckay) [inline static/js/common/upload-addon.js]: Turns out we shouldn't really be using `_blank`.

Comment (EnTeQuAk) [inline static/js/common/upload-addon.js]: indeed, there are other places too that use it. Maybe i'll try to fix them all (or most of them) in a separate PR

Comment (EnTeQuAk) [inline static/js/common/upload-addon.js]: Thanks for the PR! :laughing:

**Pull request merged**